app/src/cli/commands/clone.ts

```diff
@@ -0,0 +1,72 @@
+import * as QueryString from 'querystring'
+import { URL } from 'url'
+
+import { CommandError } from '../util'
+import { openDesktop } from '../open-desktop'
+import { ICommandModule, mriArgv } from '../load-commands'
+
+interface ICloneArgs extends mriArgv {
+  pr?: number
+  branch?: string
+}
+
+const command: ICommandModule = {
+  command: 'clone <url|slug> [path]',
+  description: 'Clone a repository',
+  args: [
+    {
+      name: 'url|slug',
+      required: true,
+      description: 'The URL to clone, or the GitHub repo slug to clone',
+      type: 'string',
+    },
+    {
+      name: 'path',
+      required: false,
+      description: 'The path to clone to',
+      type: 'string',
+    },
+  ],
+  options: {
+    branch: {
+      type: 'string',
+      aliases: ['b'],
+      description: 'The branch to switch to after cloning',
+    },
+    pr: {
+      type: 'number',
+      aliases: ['p'],
+      description: 'The PR branch to switch to after cloning',
+    },
+  },
+  handler({ _: [cloneUrl, path], pr, branch }: ICloneArgs) {
+    if (!cloneUrl) {
+      throw new CommandError('Clone URL must be specified')
+    }
+    try {
+      const _ = new URL(cloneUrl)
+      _.toString() // don’t mark as unused
+    } catch (e) {
+      // invalid URL, assume a GitHub repo
+      cloneUrl = `https://github.com/${cloneUrl}`
+    }
+    if (pr) {
+      branch = 'pr/' + pr
```

Comment:
This isn't quite right... while the PR ref will exist, in _most_ cases we don't want to check it out. The only times we want to check out the PR ref are for PRs which are coming from forks.